### PR TITLE
Pr/rosapps fix 64bit issues

### PR DESCRIPTION
**Pull request by tkreuzer**
## Purpose
Fix a bunch of x64 warnings in rosapps

### CODE REVIEW

File: modules/rosapps/applications/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 add_subdirectory(cmdutils)
 add_subdirectory(devutils)
-add_subdirectory(explorer-old)
-add_subdirectory(fraginator)
+if (${ARCH} MATCHES "i386")
```
Comment (ThFabba) [inline modules/rosapps/applications/CMakeLists.txt]: `ARCH STREQUAL "i386"`

File: modules/rosapps/applications/sysutils/CMakeLists.txt [hunk 2]
```diff
@@ -6,7 +6,9 @@ add_subdirectory(kill)
 add_subdirectory(logevent)
 add_subdirectory(lsdd)
 add_subdirectory(man)
-add_subdirectory(pedump)
+if (${ARCH} MATCHES "i386")
```
Comment (ThFabba) [inline modules/rosapps/applications/sysutils/CMakeLists.txt]: Prefer STREQUAL

File: modules/rosapps/applications/sysutils/ctm/ctm.c [hunk 3]
```diff
@@ -189,7 +189,8 @@ void DisplayScreen()
 		lines = ProcPerScreen;
 	for (idx=0; idx<ProcPerScreen; idx++)
 	{
-		int len, i;
+		int i;
+        SIZE_T len;
```
Comment (ThFabba) [inline modules/rosapps/applications/sysutils/ctm/ctm.c]: Indent seems broken here

File: modules/rosapps/applications/sysutils/CMakeLists.txt [hunk 4]
```diff
@@ -6,7 +6,9 @@ add_subdirectory(kill)
 add_subdirectory(logevent)
 add_subdirectory(lsdd)
 add_subdirectory(man)
-add_subdirectory(pedump)
+if (ARCH STREQUAL "i386")
+    add_subdirectory(pedump)
+endif()
```
[the remark this answers is not in the document]
Comment (tkreuzer) [reply modules/rosapps/applications/sysutils/CMakeLists.txt]: I don't think that depends on portability, but rather on usefulness. That is questionable with other apps as well.

**Pull request merged**